### PR TITLE
IN-27 feat: add tabs component

### PR DESCRIPTION
**Pull request by Abrosimov96**

### CODE REVIEW

File: src/components/tabs/tabs.module.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+@use "../../styles/abstracts/index" as *;
+
+.root {
```
Comment (vitaliiRss) [inline src/components/tabs/tabs.module.scss:3]: I guess here better to use `tabs` instead of `root`

File: src/components/tabs/tabs.module.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  width: 100%;
+}
+
+.list {
+  display: inline-flex;
+  flex-shrink: 0;
+  background-color: inherit;
+}
+
+.trigger {
```
Comment (vitaliiRss) [inline src/components/tabs/tabs.module.scss:15]: I guess here better to use `tab` instead of `trigger`

File: src/styles/base/_colors.scss [hunk 3]
```diff
@@ -41,4 +41,8 @@
   --color-light-500: #edf3fa;
   --color-light-700: #d5dae0;
   --color-light-900: #8d9094;
+
+  /* tabs */
+  --color-primary-active-tab: #73a5ff15;
```
Comment (vitaliiRss) [inline src/styles/base/_colors.scss:46]: Hex color should be 6 characters, but here 8. Please double check values

File: src/styles/abstracts/_tokens.scss [hunk 4]
```diff
@@ -1,6 +1,6 @@
 /* outlines */
 $outline-color: var(--color-accent-500);
-$outline-focus: 2px solid var(--outline-color);
+$outline-focus: 2px solid $outline-color;
```
Comment (vitaliiRss) [inline src/styles/abstracts/_tokens.scss:3]: I guess here is huge nesting (css variable > scss variable > another scss variable). Please use `$outline-focus: 2px solid var(--color-accent-500)`;

File: src/styles/base/_colors.scss (hunk 3 above)
Comment (Abrosimov96) [reply src/styles/base/_colors.scss:46]: Here i want to have color with transparent. And for color we decide to use css variables

File: src/components/tabs/tabs.module.scss (hunk 1 above)
Comment (Abrosimov96) [reply src/components/tabs/tabs.module.scss:3]: fix

File: src/components/tabs/tabs.module.scss (hunk 2 above)
Comment (Abrosimov96) [reply src/components/tabs/tabs.module.scss:15]: fix

File: src/styles/abstracts/_tokens.scss (hunk 4 above)
Comment (Abrosimov96) [reply src/styles/abstracts/_tokens.scss:3]: fix

**Review by MaksimSinicbIn — APPROVED** [review]

**Review by web-serge — APPROVED** [review]

**Review by SergFediaev — APPROVED** [review]

**Pull request merged**